qcengine/programs/torchani.py

```diff
@@ -73,10 +73,11 @@ def compute(self, input_data: 'ResultInput', config: 'JobConfig') -> 'Result':
 
         # Check if existings and version
         self.found(raise_error=True)
-        if parse_version(self.get_version()) < parse_version("0.5"):
-            raise ResourceError("QCEngine's TorchANI wrapper requires version 0.5 or greater.")
+        if parse_version(self.get_version()) < parse_version("0.9"):
+            raise ResourceError("QCEngine's TorchANI wrapper requires version 0.9 or greater.")
```

Comment:
should the version check happen for testing, too? https://github.com/MolSSI/QCEngine/blob/master/qcengine/testing.py#L121-L126

Reply:
Yes, you are right, I missed that. Fixed it in 7d16e98